crates/pathfinder/src/storage/state.rs

```diff
@@ -16,6 +16,17 @@ use crate::{
 /// Contains the [L1 Starknet update logs](StateUpdateLog).
 pub struct L1StateTable {}
 
+pub enum L1TableBlockId {
+    Number(StarknetBlockNumber),
+    Latest,
+}
```

Comment:
I don't really like how I've named these. But at the same time I think this whole storage could use a good cleaning up, so I didn't spend too long thinking about it.

Reply:
If it works, let's just leave it for now and rework/reword later :+1: